parser/sql/dialect/oracle/src/main/antlr4/imports/oracle/BaseRule.g4

```diff
@@ -63,7 +63,9 @@ nullValueLiterals
     ;
 
 identifier
-    : IDENTIFIER_ | unreservedWord
```

Comment:
Please keep these branches in one line. 

You can refer code conduct - https://shardingsphere.apache.org/community/en/involved/conduct/code/.

> If a rule’s branch is over than 5, every branch take a new line.